docs/bold-bi-auto-deployment.md

```diff
@@ -0,0 +1,982 @@
+            </tr>
+        </table>
+        <br/>
+        <details>
+          <summary>
+            <span style="color:blue"> Example for above environment variables:</span>  
+          </summary>
+
+            apiVersion: v1
+            kind: ConfigMap
+            metadata:
+              name: branding-config
+              namespace: bold-services
+            # All the branding images variables are accepted as URL.
+            # Image type: png, svg, jpg, jpeg
+            # Ex: https://example.com/loginlogo.jpg
+            data:
+              # This is the header logo for the application, and the preferred image size is 40 x 40 pixels.
+              <span style="color:yellow">BOLD_SERVICES_BRANDING_MAIN_LOGO: "https://i.postimg.cc/FRyqDKPT/branding-header-logo.png"</span>
+              
+              # This is the login logo for the application, and the preferred image size is 200 x 40 pixels.
+              BOLD_SERVICES_BRANDING_LOGIN_LOGO: "https://i.postimg.cc/FRyqDKPT/branding-header-logo.png"
+              
+              # This is an email logo, and the preferred image size is 200 x 40 pixels.
+              BOLD_SERVICES_BRANDING_EMAIL_LOGO: "https://i.postimg.cc/FRyqDKPT/branding-header-logo.png"
+              
+              # This is a favicon, and the preferred image size is 40 x 40 pixels.
+              BOLD_SERVICES_BRANDING_FAVICON: "https://i.postimg.cc/FRyqDKPT/branding-header-logo.png "
+              
+              # This is powered by the logo, and the preferred size is 100 x 25 pixels.
+              BOLD_SERVICES_BRANDING_FOOTER_LOGO: "https://i.postimg.cc/FRyqDKPT/branding-header-logo.png "
+              
+              # This is organization name.
+              BOLD_SERVICES_SITE_NAME: "Autodeployment"
+              
+              # This is site identifier, and it will be the part of the application URL.
+              BOLD_SERVICES_SITE_IDENTIFIER: "branding"
+
+              # Note: If you want to use the custom branding, provide the value for all branding variables 
+              # If all variable values are given, application will use the branding images,
+              # otherwise it will take the default logos.
+            ---     
+        </details>
+
+           
+    * If you need to use **Bing Map** widget feature, enter value for `widget_bing_map_enable` environment variable as `true` and API key value for `widget_bing_map_api_key`.
+       
+       ![Enable-Bingmap](images/deploy/aks/bingmap_enable.png)
+
+6. If you have a DNS to map with the application, then you can continue with the following steps, else skip to **Step 11**. 
+
+7. Uncomment the host value and replace your DNS hostname with `example.com` in deploy_aks.yaml file in line **1442**.
+
+      ![DNS](images/deploy/aks/ingress_http_yaml.png)
+
+8. If you have the SSL certificate for your DNS and need to configure the site with your SSL certificate, follow the below step or you can skip to **Step 11**.
```

Comment:
If you have the SSL certificate for your DNS and need to configure the site with it, follow these steps, otherwise, skip to **Step 11.**

Reply:
concern addressed